remill/Arch/Runtime/Operators.h

```diff
@@ -115,6 +115,16 @@ float64_t _Read(Memory *, float64_t val) {
   return val;
 }
 
+ALWAYS_INLINE static
+float32_t _Read(Memory *, In<float32_t> imm) {
+  return (float32_t)imm.val;
```

Comment:
What you want is:
`return reinterpret_cast<const float32_t &>(imm.val);`

Please remember: our style guide forbids C-style casts.